.github/workflows/moduletest-file-reset-build-release.yaml

```diff
@@ -0,0 +1,92 @@
+# Build the container image for the DPL shell on any modifications to the
+# workflow or the shell project.
+# The build is then tested, and if we where triggered by a tag, a tagged
+# version of the image is pushed to a public registry.
+
+on:
+  pull_request:
+    paths:
+      - "/infrastructure/images/moduletest-file-reset-jobs/**"
+      - ".github/workflows/moduletest-file-reset-build-release.yaml"
+    tags:
+      - "mfr-*-rc"
+  push:
+    branches:
+      - main
+    # Limit pushes on main to only build if these files has changed.
+    paths:
+      - "/infrastructure/images/moduletest-file-reset-jobs/**"
+      - ".github/workflows/moduletest-file-reset-build-release.yaml"
+    tags:
+      - "mfr-*"
+
+name: Build, test and publish Moduletest File Reset Job image
+jobs:
+  lrc_build:
+    name: Build, test and publish Moduletest File Reset Job image
+    runs-on: ubuntu-latest
+    permissions:
+      # Allow push to ghcr
+      contents: write
+      packages: write
+    # Set some environment vars about the image and container registry.
+    env:
+      # Registry to push the image to.
+      repo: ghcr.io
+      # Complete registry url for image. (omitting image name).
+      image-url: ghcr.io/danskernesdigitalebibliotek/dpl-platform/moduletest-file-reset-job
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4.2.2
+      - name: Install Task
+        uses: arduino/setup-task@v2
+      - name: Log in to the Container registry
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      # Setup QEMU and buildx so that we can build multiarch images.
```

Comment:
Why are we building multiarch images?

Reply:
What do you mean? Do we need it, or not? :)

Reply:
https://github.com/danskernesdigitalebibliotek/dpl-platform/pull/634/commits/dfe97c0489405b405920a26ee8e2866760d9554e